docs/release-notes/set_gc_policy_fix.txt

```diff
@@ -0,0 +1,5 @@
+:orphan:
+
+** Bug Fixes ** 
+
+- Command: Fixed a bug where ''det e set gc-policy'' would fail when deserializing an api response because it wasn't adjusted for the new format". 
```

Comment:
Use the back tick quotes so that rst prints it like code.. sorry I was typing from iOS and couldn't find them